net.xm1math.Texmaker.yaml

```diff
@@ -0,0 +1,41 @@
+app-id: net.xm1math.Texmaker
+runtime: org.kde.Platform
+runtime-version: '5.14'
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.texlive
+command: texmaker
+rename-desktop-file: texmaker.desktop
+rename-appdata-file: texmaker.appdata.xml
+rename-icon: texmaker
+finish-args:
+  - --socket=fallback-x11
+  - --share=ipc
+  - --socket=wayland
```

Comment:
Just curious if tested on Wayland, common issues there for Qt.

Reply:
Not tested on Wayland. Should I remove it and change to socket=x11?

Reply:
It would be safer to remove it than assume it works.

Reply:
(Going to test it though.)

Reply:
Bunch of warnings like this:

```
libEGL warning: wayland-egl: could not open /dev/dri/card0 (No such file or directory)
qt.qpa.wayland: Wayland does not support QWindow::requestActivate(
```

But seems to work fine.

Reply:
At least one of those warnings mean you need to add the `--device=dri` permission.

Reply:
done